docs/vector.rst

```diff
@@ -13,25 +13,25 @@ In fact, ``Vector`` is simply an alias of ``Matrix``, just with ``m = 1``. See :
 Declaration
 -----------
 
-As global fields of vectors
+As global vector fields
 +++++++++++++++++++++++++++
```

Comment:
Please fix al the size issues, **not only here**:
```suggestion
++++++++++++++++++++++++
```

Reply:
It's my fault, sorry.